pom.xml

```
@@ -33,6 +33,7 @@
 
     <modules>
         <module>apm-protocol</module>
+        <module>distribution</module>
```

```suggestion
        <module>native-dist</module>
```

This seems a better name. And I want to ask why this module is not controlled by the profile, instead, you added the profile inside the pom.xml of this new module?
AFAIK, if I don't want the native image, this module is not necessary too.

---

I included the distribution module because I think that perhaps one day we may need more than just one packaging way like native-image. We might have many other packaging ways, and they should be parallel to the graal module.

---

There is no other package mode. Let's not design things unexisting, OK?